gen/schema/schema.yaml

```diff
@@ -7,7 +7,8 @@ rest_endpoint: str(required=False) # REST endpoint path
 put_create: bool(required=False) # Set to true if the PUT request is used for create
 no_update: bool(required=False) # Set to true if the PUT request is not supported
 no_delete: bool(required=False) # Set to true if the DELETE request is not supported
-data_source_name_query: bool(required=False) # Set to true if the data source supports name queries
+data_source_name_query: bool(required=False) # Set to true if the data source supports name queries ### Equal to "query_parameter: name"
```

Comment:
Let's not have two ways to achieve the same thing. If we introduce a more flexible option which makes this option obsolete, let's remove it.

Reply:
That is the goal, however I wanted to take a step in between, where we support both to make sure there is an agreement on how this should be implemented (before re-doing all YAML definitions).

Reply:
Implemented for all resources